spec/blueprints/user_blueprint_spec.rb

```diff
@@ -118,7 +118,10 @@
                service_day: service_day,
                child_approval: child.child_approvals.first)
         2.times do |idx|
-          service_day = create(:service_day, child: child)
+          service_day = create(:service_day,
+                               child: child,
+                               date: \
+                                           Time.current.in_time_zone(child.timezone).at_beginning_of_day + idx.days)
```

Comment:
```suggestion
          service_day = create(
            :service_day,
            child: child,
            date: Time.current.in_time_zone(child.timezone).at_beginning_of_day + idx.days
          )
```
Let's format this way instead of using `\` for line breaks